.env.exaple

```diff
@@ -1 +1,2 @@
 SKIP_PREFLIGHT_CHECK=true
+BASE_URL
```

Comment:
Please, add example value that points to localhost

Reply:
done